packages/manager/apps/pci-vouchers/src/components/vouchers/listing/Product.spec.tsx

```diff
@@ -10,14 +10,12 @@ vi.mock('@ovhcloud/manager-components', async () => ({
 
 vi.mock('react-i18next', () => ({
   // this mock makes sure any components using the translate hook can use it without a warning being shown
-  useTranslation: () => {
-    return {
-      t: (str: string) => str,
-      i18n: {
-        changeLanguage: () => new Promise(() => {}),
-      },
-    };
-  },
+  useTranslation: () => ({
+    t: (str: string) => str,
+    i18n: {
+      changeLanguage: () => new Promise(() => {}),
+    },
+  }),
```

Comment:
already defined in `packages/manager/apps/pci-vouchers/src/setupTests.ts` so no need to define it one more time

Reply:
done